mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,18 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+
+function introduceMe(name, age) {
+  return "Hello, my name is " + name + " and I am " + age + "years old";
```

Comment:
Looks like you have a minor bug here, just missing a space before "years old". Also, this may be a nice place to try out _string interpolation_ - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals